### ISSUE TITLE
"Choose representative" page is missing a "Write to [MP name here]" link

### ISSUE THREAD

**Issue by zarino** [mysociety/writetothem#297]
There's a "Write to all your councillors" link, and a "Write to all your MEPs" link, but not a "Write you [your MP]" link.

![screen shot 2016-01-25 at 09 54 00](https://cloud.githubusercontent.com/assets/739624/12547726/ddf0abfa-c349-11e5-872d-b00d29c9914f.png)

If people miss don't realise that they're meant to click the recipient’s name, the only clear calls to action (ie: "[Verb] [object]") on the page, are the "Write to all your councillors" and "Write to all your MEPs" links.

There's no corresponding "Write to your MP" link.

**Comment by MyfanwyNixon** [comment]
Yes, I think this is partly what has caused a cross email to user support today.

> Your system [...] uses the 'blunderbuss' approach to send e-mails to ALL MEP's in my area, which happens to be in the Eastern Region.

(The user's wider point was incorrect, that he wanted to contact only the MEP 'for his constituency' and he wanted us to point out which one that was, but part of the issue certainly seemed to be that he hadn't noticed you could mail each MEP separately)

**Comment by zarino** [comment]
I was operating on autopilot a minute ago, and totally got caught out by this. I saw the "Write to all your councillors" link first, then scanned down the column, expecting to find a similar link for my MP. But instead, all I found was the outbound TWFY link. And I didn't know what to do.

**Comment by MyfanwyNixon** [comment]
Not sure but it's possible this person is experiencing the same issue:

>  I would like to congratulate contact with my local mp
     but I don't know how. Any phone numbers? Because this site
     doesn't show me the way to make a contact
